modules/490-virtualization/templates/docs/CONFIGURATION.md

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
why the docs dir in templates?

Reply:
delete it

Reply:
fixed in da0667eff2eb6dfc02c4ea684c126ecb690a1b1d